### ISSUE TITLE
WeatherUnderground No Longer offers free API

### ISSUE THREAD

**Issue by FrankLaVigne** [ZoinerTejada/mcw-big-data-and-visualization#1]
Looks like the free API offering of Wunderground is gone.

https://www.wunderground.com/weather/api/

And the sign up link for more information implies you need $250k minimum of annual revenue to even be worthy of a response from Big Blue. 

What are the plans to work around this?

**Comment by joelhulen** [comment]
@FrankLaVigne We were notified of this issue last week. We've already found a suitable replacement (Dark Sky) and have updated the code to use it. That update has not been merged into this repo yet, since we are in the process of doing a major update to this package. The new weather API will be implemented in the new version of the lab. I'll update this issue with more details once the new version is available.

**Comment by FrankLaVigne** [comment]
Thanks! And I really appreciate the rapid response!

Do you have a time estimate?

**Comment by Diegofox82** [comment]
Hi @joelhulen, do you have any update on the integration of the weather API replacement?

**Comment by joelhulen** [comment]
Yes. Sorry, this is an outdated repo for this MCW. I changed it over to Darksky last year. Please use the currently maintained version of this repo here: https://github.com/Microsoft/MCW-Big-Data-and-Visualization
